apps/next/src/pages/AccountManagement/components/ChainListItem.tsx

```diff
@@ -25,9 +26,13 @@ export type Props = {
   action?: ReactElement;
 };
 
-const ItemIcon = styled(ListItemIcon)({
+const ItemIcon = styled(ListItemIcon)(({ theme }) => ({
```

Comment:
If we're using this component outside of `pages/AccountManagement`, let's move the file out to `src/components/Address` as well, similar to `AddressItem` I mentioned above.